pkg/sqlite/load.go

```diff
@@ -805,3 +805,49 @@ func (s *sqlLoader) addBundleProperties(tx *sql.Tx, bundle *registry.Bundle) err
 
 	return nil
 }
+
+func (s *sqlLoader) rmChannelEntry(tx *sql.Tx, csvName string) error {
+	stmt, err := tx.Prepare("DELETE FROM channel_entry WHERE operatorbundle_name=?")
```

Comment:
can we use version and bundlepath here too, just to be safe?

Reply:
talked offline with @ecordell about this. If / when the csv name stops being unique we will make the change